.changeset/purple-pans-shout.md

```diff
@@ -0,0 +1,35 @@
+---
+"@atproto/lex-cli": minor
+"@atproto/xrpc": minor
+"@atproto/api": minor
+"@atproto/xrpc-server": patch
```

Comment:
There are no changes to xrpc-server— it will get a patch by virtue of its dependencies being updated.
```suggestion
```